analyzers/src/SonarAnalyzer.Common/Common/UnexpectedLanguageException.cs

```diff
@@ -23,5 +23,7 @@ namespace SonarAnalyzer.Common
     public sealed class UnexpectedLanguageException : Exception
     {
         public UnexpectedLanguageException(AnalyzerLanguage language) : base($"Unexpected language: {language}") { }
+
+        public UnexpectedLanguageException(string message) : base(message) { }
```

Comment:
Find a difference between this constructor and the other one. Keep the purpose of the class in mind. And remove duplications from the code.

Reply:
@pavel-mikula-sonarsource  Does it make sense that I just change the first constructor to accept a string and change the calls to get the `Language.LanguageName`?

Reply:
Almost. I would not swap their order. In the original order, the value gets propagated down and processed.